WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
 			}
 			is_adult = yes
-			prestige = 5000
+			prestige = 2000
```

Comment:
Again these arbitrary changes to prestige requirements. I told you this change didn't add much to the mod and was entirely random with values that didn't correspond overall, so I recommend scrapping it.